.github/workflows/ci-test.yaml

```diff
@@ -0,0 +1,132 @@
+    - name: Setup UTF-8 Capable locale
+      run: | 
+        export LC_ALL=C.UTF-8
+        export LANG=C.UTF-8
+        export LANGUAGE=C.UTF-8
+    - name: Setup Configuration Files
+      run: |
+        cp config.js.example config.js
+        cp config.js.example test/config.js
+        cp config-build.js.example config-build.js
+        cp config-server.js.example config-server.js
+        cp test/config.yml.example test/config.yml
+  
+    - name: Ngrok
+      run: |
+        curl -s https://ngrok-agent.s3.amazonaws.com/ngrok.asc | sudo tee /etc/apt/trusted.gpg.d/ngrok.asc >/dev/null && echo "deb https://ngrok-agent.s3.amazonaws.com buster main" | sudo tee /etc/apt/sources.list.d/ngrok.list && sudo apt update && sudo apt install ngrok
+        export GITHUB_TAG=0.0.0
+      
+    - name: Setup Pender 
+      run: | 
+        export PENDER_BRANCH=$((git ls-remote --exit-code --heads https://github.com/meedan/pender.git $GITHUB_BRANCH >/dev/null && echo $GITHUB_BRANCH) || echo 'develop')
+        echo "Using branch $PENDER_BRANCH for Pender"
+        git clone --branch=$PENDER_BRANCH https://github.com/meedan/pender.git
+        cd pender
+        cp config/config.yml.example config/config.yml
+        cp config/database.yml.example config/database.yml
+        cp config/sidekiq.yml.example config/sidekiq.yml
+        cd -
+  
+    - name: Setup Check API
+      run: | 
+        export CHECK_API_BRANCH=$((git ls-remote --exit-code --heads https://github.com/meedan/check-api.git $GITHUB_BRANCH >/dev/null && echo $GITHUB_BRANCH) || echo 'develop')
+        echo "Using branch $CHECK_API_BRANCH for Check API"
+        git clone --branch=$CHECK_API_BRANCH https://github.com/meedan/check-api.git
+        cd check-api
+        cp config/config.yml.example config/config.yml
+        cp config/database.yml.example config/database.yml
+        cp config/sidekiq.yml.example config/sidekiq.yml
+        cd -    
+        
+    - name: Setup Alegre
+      run: | 
+        export ALEGRE_BRANCH=$((git ls-remote --exit-code --heads https://github.com/meedan/alegre.git $GITHUB_BRANCH >/dev/null && echo $GITHUB_BRANCH) || echo 'develop')
+        echo "Using branch $ALEGRE_BRANCH for Alegre"
+        git clone --branch=$ALEGRE_BRANCH https://github.com/meedan/alegre.git
+        cd alegre
+        cp .env_file.test .env_file
+        cp ../test/google_credentials.json google_credentials.json
+        cd -
+  
+    - name: Setup Presto
+      run: |
+        export PRESTO_BRANCH=$((git ls-remote --exit-code --heads https://github.com/meedan/presto.git $GITHUB_BRANCH >/dev/null && echo $GITHUB_BRANCH) || echo 'master')
+        echo "Using branch $PRESTO_BRANCH for Presto"
+        git clone --branch=$PRESTO_BRANCH https://github.com/meedan/presto.git
+        cd presto
+        cp .env_file.test .env_file
+        cp ../test/google_credentials.json google_credentials.json
+        cd -
```

Comment:
None of this is needed if you're trying to just run the linter and/or unit tests... so, it could be out of this first PR, for simplicity sake and to get something up and running faster.

Reply:
Removed these and the build failed 
```
Run docker compose exec web npm run linter
env file /home/runner/work/check-web/check-web/presto/.env_file not found: stat /home/runner/work/check-web/check-web/presto/.env_file: no such file or directory
Error: Process completed with exit code 14.
```

Reply:
Ah, I think this is because they are still referenced in `docker-compose.yml`...

Reply:
Yeah, so I guess you'll need to at least pull the code and configuration files.

Reply:
yeah. I will add these back